Lib/_pyrepl/unix_console.py

```diff
@@ -561,6 +561,11 @@ def clear(self):
         self.posxy = 0, 0
         self.screen = []
 
+    def clear_all(self) -> None:
+        """Clear screen and scrollback buffer."""
+        self.__write("\x1b[3J\x1b[2J\x1b[H")
```

Comment:
```suggestion
        self.__write("\x1b[H\x1b[3J\x1b[2J")
```
Altough both worked for me on Ubuntu 24.0.4.3 LTS, maybe starting with `\x1b[H` is better?
Since it is used like so for clear
https://github.com/python/cpython/blob/1c984ba953e140b88ff890dbe1408f9611027444/Lib/_pyrepl/terminfo.py#L176
https://github.com/python/cpython/blob/1c984ba953e140b88ff890dbe1408f9611027444/Lib/_pyrepl/terminfo.py#L250
And for symmetry with Windows (see below)?